doc/march_packages/march_input_device.rst

```diff
@@ -71,25 +71,23 @@ Start the software
     roslaunch march_launch serial_connection.launch tcp:=false
 
 
-How to run Wireless
+How to run wireless
 ^^^^^^^^^^^^^^^^^^^
 
-.. todo:: (Electro) Document how to change the input device to wireless mode
-
 .. note:: Make sure the pc that needs to receive the messages and the input device are on the same network.
 
 Upload the code on the input device
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-Skip this section if the correct code for running wired is already on the input device. More detailed information on how
+Skip this section if the correct code for running wireless is already on the input device. More detailed information on how
 to run and upload the code see the :input-device:`ReadMe <>`.
 
-.. note:: If you want to change the network name and password that the input device uses: Change **ssid** and **password** in the *WirelessConnection.h*
-
 - Type in terminal: :code:`hostname -I` Remember the output, this is the hostname. You have to use this address for the next instructions. From this point we call this value **your_hostname**.
 - Uncomment :code:`#define USE_WIRELESS` in the *main.cpp*.
-- Change :code:`IPAddress server(x, x, x, x)` to :code:`IPAddress server(your_hostname)` in the *WirelessConnection.h*
+- Change :code:`IPAddress server(x, x, x, x)` to :code:`IPAddress server(your_hostname)` in the *WirelessConnection.h*.
 - Upload to input device.
 
+.. note:: If you want to change the network name and password that the input device uses: Change **ssid** and **password** in the *WirelessConnection.h*
```

Comment:
Link to the actual file

Reply:
:input-device:`WirelessConnection.h <link/to/WirelessConnection>`.

Reply:
Old documentation, I just moved it. Probably better to open a new issue for it.

Reply:
I've added it